src/audio/index.ts

```diff
@@ -0,0 +1,15 @@
+export {
+	AudioPlayer,
+	AudioPlayerStatus,
+	NoSubscriberBehaviour,
+	createAudioPlayer
+} from './AudioPlayer';
+
+export {
+	AudioResource,
+	createAudioResource
+} from './AudioResource';
+
+export { PlayerSubscription } from './PlayerSubscription';
```

Comment:
```suggestion
export * from './AudioPlayer';
export * from './AudioResource';
export * from './PlayerSubscription';
```
All these files have the same amount of exports as what you're exporting manually, so you can use the wildcard here.

Reply:
Yep, I've chosen to explicitly export these things to avoid accidentally exporting private types in the future.